core/pkg/resultshandling/printer/v2/controltable.go

```diff
@@ -35,7 +35,11 @@ func generateRow(controlSummary reportsummary.IControlSummary, infoToPrintInfo [
 	}
 
 	row[columnSeverity] = getSeverityColumn(controlSummary)
-	row[columnName] = controlSummary.GetName()
+	if len(controlSummary.GetName()) > 50 {
```

Comment:
I think @fredbi was also implementing something re pretty printing

Reply:
This is not a great solution, in generate we'd need to take into account the terminal with and etc.
Maybe we should look at an existing table implementation as well

Reply:
https://github.com/kubescape/kubescape/issues/933